<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SaveMaskObjDialog</class>
 <widget class="QDialog" name="SaveMaskObjDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>90</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>0</y>
     <width>95</width>
     <height>78</height>
    </rect>
   </property>
   <property name="title" >
    <string>Type Save</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QRadioButton" name="binaryButton" >
      <property name="text" >
       <string>Binary Obj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="asciiButton" >
      <property name="text" >
       <string>ASCII Obj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Options Save</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QCheckBox" name="vertexCheck" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Vertexs</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QCheckBox" name="colorVertexCheck" >
      <property name="text" >
       <string>Color Faces</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QCheckBox" name="textureCheck" >
      <property name="text" >
       <string>Texture Coords</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QCheckBox" name="faceCheck" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Faces</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QCheckBox" name="normalCheck" >
      <property name="text" >
       <string>Normals Vertex</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SaveMaskObjDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SaveMaskObjDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
